corehq/pillows/mappings/user_mapping.py

```diff
@@ -95,7 +96,20 @@
                                        },
                                        'type': 'object'},
                 'status': {'type': 'string'},
-                'user_data': {'type': 'object', 'enabled': False},
+                'user_data_es': {
+                    'type': 'nested',
+                    'dynamic': False,
+                    'properties': {
+                        'key': {
+                            'type': 'string',
+                            'index': 'not_analyzed',
+                        },
+                        'value': {
+                            'type': 'string',
+                            'index': 'not_analyzed',
```

Comment:
not sure why you wouldn't leave this as being analyzed for if we want to run real queries on these values in the future. I guess we can just rewrite the mapping at that point?